github/VisualStudio
# Show Team Explorer Home before starting to clone

_Note: This is made obsolete by #1982_

We need to make sure Team Explorer - Home is visible before the clone operation starts so the user can see progress have the option to cancel.

### What this PR does

- Show `Team Explorer - Home` before clone starts rather than after it finishes

Make sure this UI is visible when cloning:

![image](https://user-images.githubusercontent.com/11719160/46607790-308d4700-cafa-11e8-8647-e1f05e5aa968.png)

### How to test

1. Open the Clone dialog
2. Start cloning a repository (e.g. https://github.com/dotnet/roslyn)
3. Check `Team Explorer - Home` / `Clone Repository` is visible

Fixes #1977
